streampipes-client-python/streampipes/endpoint/api/version.py

```diff
@@ -109,11 +109,20 @@ def get(self, identifier: str, **kwargs) -> Resource:
         identifier: str
             Not supported by this endpoint, is set to an empty string.
 
+        Raises
+        ------
+        NotImplementedError
+            Non-empty 'identifier' is not supported by this endpoint. Please set 'identifier' to an empty string.
+
         Returns
         -------
         versions: Version
             The specified resource as an instance of the corresponding model class([Version][streampipes.model.resource.Version]).  # noqa: 501
         """
+        if identifier != "":
```

Comment:
Shall we change that to `if identifier:`? This would allow the user to pass `None` as well which is semantically correct.